node_groups.tf

```diff
@@ -150,6 +142,15 @@ locals {
       type                          = "ingress"
       source_cluster_security_group = true
     }
+    # VPC CNI
+    ingress_cluster_vpc_cni_metrics = {
+      description                   = "Cluster API to node 61678/tcp VPC CNI metrics"
+      protocol                      = "tcp"
+      from_port                     = 61678
```

Comment:
The "recommended" rules are those that are most commonly found in clusters - there isn't a set rule of what makes it "common" or "recommended", but based off of historical issues raised by users. The VPC CNI metrics and its associated access/port has not come up so I don't believe this falls under the recommended category. 

If a user wishes to use this addon and open up this network connectivity, they can do so by providing the rule in their configuration similar to shown here https://github.com/terraform-aws-modules/terraform-aws-eks/blob/674ab7b7f2aa5163b3d9dbdafb7cb1ec910cd9de/examples/complete/main.tf#L78-L117